Compiler/README.md

```diff
@@ -65,5 +65,7 @@ To copy the executable out of the docker container, use:
         - Classes
         - Spread (update Fuzzilli)
     - Improve for/while loops. The current implementation makes all loops while loops, with a comparison against 0.
+    - Improve for-of/for-in loops. A large number of test cases have more complex left sides, while Fuzzilli only supports declaring/defining a new variable.
+        - Example from a regression: for (a[i++].x in [])
```

Comment:
Oh interesting, I'm not even sure I was aware of that kind of JS syntax...

Reply:
Me neither, until I ran into samples that wouldn't compile! I'm not sure there's much value in compiling such samples. Thoughts? Maybe this should be done as a main thread discussion.

Reply:
Yeah I agree. I think it's good the way it is now, i.e. just mentioning it in the README as TODO.